Modules/_testexternalinspection.c

```diff
@@ -322,10 +333,13 @@ get_py_runtime_linux(pid_t pid)
     Elf_Shdr* shstrtab_section = &section_header_table[elf_header->e_shstrndx];
     char* shstrtab = (char*)(file_memory + shstrtab_section->sh_offset);
 
-    Elf_Shdr* py_runtime_section = NULL;
+    Elf_Shdr* section = NULL;
     for (int i = 0; i < elf_header->e_shnum; i++) {
-        if (strcmp(".PyRuntime", shstrtab + section_header_table[i].sh_name) == 0) {
-            py_runtime_section = &section_header_table[i];
+        char* this_sec_name = shstrtab + section_header_table[i].sh_name;
+        // Move 1 character to account for the leading "."
+        this_sec_name += 1;
```

Comment:
can't we just have the `+1` in the above expression and use a `const char *` instead?

Reply:
I guess this is split to explain what the "+1" is exactly.

Reply:
This is a test code, I wouldn't obsess over it.

Reply:
Since I've reformatted the entire file I'll fix this nit. Using `const` is always a good idea when we can.